static/app/views/settings/organizationDeveloperSettings/permissionSelection.spec.jsx

```diff
@@ -27,74 +30,45 @@ describe('PermissionSelection', () => {
-      'No Access',
-      'Read',
-      'Read & Write',
-      'Admin',
-    ]);
-    expect(findOptions('Member')).toEqual(['No Access', 'Read', 'Read & Write', 'Admin']);
-  });
-
-  it('converts permission state to a list of raw scopes', () => {
-    const instance = wrapper.find('PermissionSelection').instance();
-    instance.setState({
-      permissions: {
-        Project: 'write',
-        Release: 'admin',
-        Organization: 'read',
-      },
-    });
-
-    expect(instance.permissionStateToList()).toEqual([
-      'project:read',
-      'project:write',
-      'project:releases',
-      'org:read',
-    ]);
+    await expectOptions('Project', ['No Access', 'Read', 'Read & Write', 'Admin']);
+    await expectOptions('Team', ['No Access', 'Read', 'Read & Write', 'Admin']);
+    await expectOptions('Release', ['No Access', 'Admin']);
+    await expectOptions('Issue & Event', ['No Access', 'Read', 'Read & Write', 'Admin']);
+    await expectOptions('Organization', ['No Access', 'Read', 'Read & Write', 'Admin']);
+    await expectOptions('Member', ['No Access', 'Read', 'Read & Write', 'Admin']);
   });
 
-  it('stores the permissions the User has selected', () => {
-    const instance = wrapper.find('PermissionSelection').instance();
-    const getStateValue = resource => instance.state.permissions[resource];
+  it('stores the permissions the User has selected', async () => {
+    const selectByValue = (name, value) =>
+      selectEvent.select(screen.getByRole('textbox', {name}), value);
 
-    selectByValue(wrapper, 'write', {name: 'Project--permission'});
-    selectByValue(wrapper, 'read', {name: 'Team--permission'});
-    selectByValue(wrapper, 'admin', {name: 'Release--permission'});
-    selectByValue(wrapper, 'admin', {name: 'Event--permission'});
-    selectByValue(wrapper, 'read', {name: 'Organization--permission'});
-    selectByValue(wrapper, 'no-access', {name: 'Member--permission'});
+    await selectByValue('Project', 'Read & Write');
+    await selectByValue('Team', 'Read');
+    await selectByValue('Release', 'Admin');
+    await selectByValue('Issue & Event', 'Admin');
+    await selectByValue('Organization', 'Read');
+    await selectByValue('Member', 'No Access');
 
-    expect(getStateValue('Project')).toEqual('write');
-    expect(getStateValue('Team')).toEqual('read');
-    expect(getStateValue('Release')).toEqual('admin');
-    expect(getStateValue('Event')).toEqual('admin');
-    expect(getStateValue('Organization')).toEqual('read');
-    expect(getStateValue('Member')).toEqual('no-access');
+    expect(model.getValue('Project--permission')).toEqual('write');
+    expect(model.getValue('Team--permission')).toEqual('read');
+    expect(model.getValue('Release--permission')).toEqual('admin');
+    expect(model.getValue('Event--permission')).toEqual('admin');
+    expect(model.getValue('Organization--permission')).toEqual('read');
+    expect(model.getValue('Member--permission')).toEqual('no-access');
```

Comment:
IMO you might as well leave the `model` out and assert on the value of the hidden inputs, but that's just me! ✨ integrationy ✨

Reply:
It's kind of hard with RTL to check hidden input values actually hah

Reply:
Oh interesting! Alright 👌🏻